### PR TITLE
drawer cleanup

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 3552 .

#### Changes proposed in this Pull Request:
- Favicon icon updated
- Fixed Page body should have some padding/margin when small:
image
- Fixed Tabs should not overflow (either shrink them or make them scrollable style):
image

- Removed the Getting Started NavGroup
- Removed the home page - redirect to first component for now
- Removed Menu Icon in toolbar from drawer
image
- Closed the temporary drawer on mobile

#### To Test:
-

### CODE REVIEW

File: docs/src/shared/ComponentPreviewTabs.tsx [hunk 1]
```diff
@@ -43,13 +43,14 @@ const tabStyles: any = {
         color: (theme: Theme) => theme.palette.primary.main,
     },
 };
+
 const tabPanelContentStyles: any = {
```
Comment (bkarambe) [inline docs/src/shared/ComponentPreviewTabs.tsx:47]: Please update the type to SxProps<Theme>

File: docs/src/shared/ComponentPreviewTabs.tsx [hunk 2]
```diff
@@ -43,13 +43,14 @@ const tabStyles: any = {
...
     display: 'flex',
     flexDirection: 'column',
     height: '100%',
     minHeight: (theme: Theme) => `calc(100vh - ${theme.spacing(8)})`,
-    maxWidth: '980px',
-    m: '0px auto',
+    maxWidth: '1080px',
+    m: '0 auto',
```
Comment (bkarambe) [inline docs/src/shared/ComponentPreviewTabs.tsx:53]: mx: 'auto' should work

File: docs/src/pages/componentPreviewPage.tsx [hunk 3]
```diff
@@ -24,14 +25,14 @@ export const ComponentPreviewPage: React.FC<ComponentPreviewPageProps> = (props)
                 }}
             >
                 <Toolbar sx={{ pl: 2, pr: 2 }}>
```
Comment (bkarambe) [inline docs/src/pages/componentPreviewPage.tsx:27]: instead of pl and pr use px

File: docs/src/pages/componentPreviewPage.tsx [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
 export const ComponentPreviewPage: React.FC<ComponentPreviewPageProps> = (props): JSX.Element => {
     const { title, ...divProps } = props;
+    const dispatch = useAppDispatch();
     const theme = useTheme();
-    const { setDrawerOpen } = useDrawer();
-    const md = useMediaQuery(theme.breakpoints.up('md'));
+    const lgUp = useMediaQuery(theme.breakpoints.up('lg'));
 
     return (
         <div {...divProps} style={{ display: 'flex', flexDirection: 'column' }}>
```
Comment (bkarambe) [inline docs/src/pages/componentPreviewPage.tsx:18]: Convert div to Box and style to sx

File: docs/src/router/drawer.tsx [hunk 5]
```diff
@@ -141,7 +151,6 @@ export const NavigationDrawer: React.FC = () => {
                             '.BluiDrawerNavGroup-title': {
```
Comment (bkarambe) [inline docs/src/router/drawer.tsx:151]: Not sure, this gets applied, missing **&**

File: docs/src/shared/ComponentPreviewTabs.tsx [hunk 6]
```diff
@@ -43,13 +43,14 @@ const tabStyles: any = {
         color: (theme: Theme) => theme.palette.primary.main,
```
Comment (bkarambe) [inline docs/src/shared/ComponentPreviewTabs.tsx:43]: use 'primary.main' only

File: docs/src/shared/ComponentPreviewTabs.tsx [hunk 7]
```diff
@@ -105,22 +109,71 @@ export const ComponentPreviewTabs = (): JSX.Element => {
...
                     </Box>
                 </Box>
             </TabPanel>
             <TabPanel value={value} index={1}>
-                <Box sx={tabPanelContentStyles}>
+                <Box
+                    sx={{
+                        ...tabPanelContentStyles,
+                        [theme.breakpoints.down(1440)]: {
+                            m: '0 24px',
```
Comment (bkarambe) [inline docs/src/shared/ComponentPreviewTabs.tsx:172]: mx: 3 should work

File: docs/src/shared/ComponentPreviewTabs.tsx [hunk 8]
```diff
@@ -105,22 +109,71 @@ export const ComponentPreviewTabs = (): JSX.Element => {
...
                     )}
                 </Tabs>
             </Box>
             <TabPanel value={value} index={0}>
-                <Box sx={tabPanelContentStyles}>
+                <Box
+                    sx={{
+                        ...tabPanelContentStyles,
+                        [theme.breakpoints.down(1440)]: {
+                            m: '0 24px',
```
Comment (bkarambe) [inline docs/src/shared/ComponentPreviewTabs.tsx:157]: mx: 3

**Comment by emclaug2** [comment]
Changes visually look nice - Drawer looks as expected and page content is mostly responsive.  There's a couple examples that still cause some issues on smaller screens but I don't think this is something that needs to be fixed before MVP, since 90% of users _should_ be using this on desktop.

Screen Shot 2022-10-07 at 10 27 13 AM

File: docs/src/shared/ComponentPreviewTabs.tsx (hunk 6 above)
Comment (ektasawant) [reply docs/src/shared/ComponentPreviewTabs.tsx:43]: > primary.main

how this works?

File: docs/src/router/drawer.tsx (hunk 5 above)
Comment (ektasawant) [reply docs/src/router/drawer.tsx:149]: This is working correctly.

**Comment by ektasawant** [comment]
> Changes visually look nice - Drawer looks as expected and page content is mostly responsive. There's a couple examples that still cause some issues on smaller screens but I don't think this is something that needs to be fixed before MVP, since 90% of users _should_ be using this on desktop.
> 
> Screen Shot 2022-10-07 at 10 27 13 AM

For Mobile view we need to have different story that we can check post MVP. I will log an issue for this.

**Review by huayunh — APPROVED** [review]

**Review by emclaug2 — APPROVED** [review]

File: .github/workflows/firebase-hosting-merge.yml [hunk 9]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: cd components && yarn && cd ../docs && yarn && cd .. && yarn link:components && cd docs && yarn build
-      - uses: FirebaseExtended/action-hosting-deploy@v0
+      - uses: emclaug2/action-hosting-deploy@main
```
Comment (huayunh) [inline .github/workflows/firebase-hosting-merge.yml:15]: by the way @emclaug2 is firebase settled now? Or would you recommend us to keep using your fork?

Comment (emclaug2) [reply .github/workflows/firebase-hosting-merge.yml:15]: I'm like 90% sure the firebase issues are resolved now; if @ektasawant or @bkarambe wants to test it out by reverting this line to check that'd be awesome.

Comment (ektasawant) [reply .github/workflows/firebase-hosting-merge.yml]: Revert the changes working fine 👍

**Pull request merged**